src/sleep.cc

```diff
@@ -33,11 +37,23 @@ void SleepForSeconds(double seconds) {
 }
 #else   // BENCHMARK_OS_WINDOWS
 void SleepForMicroseconds(int microseconds) {
+#ifdef BENCHMARK_OS_ZOS
+  // z/OS does not support nanosleep. Instead call sleep() and then usleep() to
+  // sleep for the remaining microseconds because usleep() will fail if its
+  // argument is greater than 1000000.
+  div_t sleepTime = div(microseconds, kNumMicrosPerSecond);
+  unsigned int seconds = sleepTime.quot;
```

Comment:
`div_t` contains two integers. you store it in `unsigned int` and then loop until it's <= 0. given it can't be <0 as it's unsigned this is not appropriate.

Please store it as an `int`.

Also as `sleep` is documented as returning explicitly `0` if there's no time left to sleep, you can check `while (seconds != 0)`.

Reply:
Thanks for reviewing. I've updated the PR to address your comments.